src/main/cluster.ts

```diff
@@ -252,16 +252,12 @@ export class Cluster implements ClusterModel, ClusterState {
   }
 
   protected async k8sRequest<T = any>(path: string, options: RequestPromiseOptions = {}): Promise<T> {
+    (options.headers ??= {}).Host = `${this.id}.${new URL(this.kubeProxyUrl).host}` // required in ClusterManager.getClusterForRequest()
```

Comment:
can be `options.headers?.Host = ${this.id}.${new URL(this.kubeProxyUrl).host}`
`(options.headers ??= {})` is not very readable for me.

Reply:
I will split `options.headers ??= {}` to a different line. Since we always want headers to be defined with at least that `Host` field.